### ISSUE TITLE
Load File API version Issue

### ISSUE THREAD

**Issue by ziadkiwan** [kenorb-contrib/tg#8]
Hello Yesterday while trying to do load _ile on any message i get the following error!
do you have any suggestion for fix ?

FAIL: 71: RPC_CALL_FAIL 400: FILEREF_UPGRADE_NEEDED
 *** 1571203415.848334 Notification API_ERROR_UPDATE_UNOFFICIAL: Error. Please update your app to the latest version or switch to the official version of Telegram.

**Comment by ziadkiwan** [comment]
Hello @kenorb  any news regarding the issue ?

**Comment by Paulchen232** [comment]
On what and how you have install it?

**Comment by ziadkiwan** [comment]
ubuntu 19.10, compiled it from source

**Comment by Paulchen232** [comment]
Ok, you have do "git clone --recursive https://github.com/CHANGETHIS/tg.git && cd tg" 
and than do `make`

**Comment by ziadkiwan** [comment]
okay let me try again (Y), and i will get back to you! thanks!

**Comment by ziadkiwan** [comment]
Now it just get stuck at 0%DOWN

**Comment by Paulchen232** [comment]
On "Raspberry PI Zero W" with vysheng/tg it works.

**Comment by ziadkiwan** [comment]
tried this, and vsheng one same issue, strange some files works and others just freeze on 0%

**Comment by gurucubano** [comment]
There is a discussion about this kind of issues here: https://github.com/vysheng/tg/issues/1629
The developer(s) of the TELEports tg app clearly state that another Telegram API has to be used to overcome this kind of issues.

**Comment by gurucubano** [comment]
There is a discussion about this kind of issues here: https://github.com/vysheng/tg/issues/1629
The developer(s) of the TELEports tg app clearly state that another Telegram API has to be used to overcome this kind of issues.

**Comment by gurucubano** [comment]
```
> view_photo 4
FAIL: 92: RPC_CALL_FAIL 400: FILEREF_UPGRADE_NEEDED
 *** 1578896852.544330 Notification API_ERROR_UPDATE_UNOFFICIAL: Error. Please update your app to the latest version or switch to the official version of Telegram.
[0%Down]>
```
this is on FreeBSD with a ```tg``` compiled from git yesterday.

**Comment by kenorb** [comment]
Btw. Alternative Telegram client: https://github.com/zevlg/telega.el

**Comment by gurucubano** [comment]
El día Monday, January 13, 2020 a las 01:31:46AM -0800, Rafal W. escribió:

> Btw. Alternative Telegram client: https://github.com/zevlg/telega.el
> 

Yes, but this is with EMACS and not a simple cmd line client.

	matthias

-- 
Matthias Apitz, ✉ guru@unixarea.de, http://www.unixarea.de/ +49-176-38902045
Public GnuPG key: http://www.unixarea.de/key.pub
May, 9: Спаси́бо освободители! Thank you very much, Russian liberators!

**Comment by psychosys** [comment]
There is no way you could fix this unless modify the code according to new telegram schema.tl
https://core.telegram.org/schema

for files and photos it requires fundamental changes ! , I see a lot of improvements the only thing remains as issue is the problem of downloading files/media that doesn't work at all 


but surprisingly sometimes it works fine because telegram allows old apps to use the previous schema .. but at this moment it is not functional
